src/internal/components/dropdown/index.tsx

```diff
@@ -31,28 +31,35 @@ import { DropdownProps } from './interfaces';
 import styles from './styles.css.js';
 
 interface DropdownContainerProps {
+  triggerRef: React.RefObject<HTMLElement>;
   children?: React.ReactNode;
-  renderWithPortal?: boolean;
+  renderWithPortal: boolean;
```

Comment:
Codecov complained that in `renderWithPortal = false` the `= false` branch is never tested. Turns out it is completely unused. Let's remove and tighten up the types